lib/parameter_popup.dart

```diff
@@ -140,7 +140,7 @@ class ParameterPopup {
 
       //update popup position
       int newLeft = math.max(
-          cursorCoords.x - (parameterHint.text.length * charWidth ~/ 2), 22);
+          cursorCoords.x - (parameterHint.text.length * charWidth ~/ 2), 22).round();
```

Comment:
If we're getting cursorCoords from the DOM, the position can be a double - a non-int value.

Since we're rounding, do we need the integer divide? (`charWidth ~/ 2`)